benchmark/cpp_xorUpdateRandom_readRandom.sh

```diff
@@ -0,0 +1,257 @@
+me=`basename $0`
+if [ $# -eq 0 ]; then
+  echo "ERROR: No db path provided."
+  echo "Usage: ${me} <path-to-db>"
+  exit 1
+fi
+
+############################################
+####   General Benchmark                ####
+############################################
+# benchmarks  -- see each benchmark
+
+num=$((1 * 1000 * 1000))
+reads=$num
+key_size=2
+value_size=$((4 * 1024))
+
+# Number of concurrent threads to run for the benchmark.
+threads=2
+
+
+############################################
+####   Statistics                       ####
+############################################
+# Report database statistics (true/false)
+statistics=1
+
+# Print histogram of operation timings (true/false)
+histogram=1
+
+# a temp varibale that estimates the total size of an entry
+entry_size=$(($key_size + $value_size))
+
+# Stats are reported every N operations when this is greater than zero.
+# When 0 the interval grows over time. The formula below comes out of an
+# experiment where we had an 100K interval with 16B keys and 8B values.
+# interval*entry_size=100K*(16+8), hence interval = 2400K/entry_size
+stats_interval=$((240000000 / $entry_size))
```

Comment:
Great work! I will add that as well. Since stats_interval_seconds overrides stats_interval, I will keep both.